agent/src/softwarecontaineragent.cpp

```diff
@@ -135,6 +147,7 @@ void SoftwareContainerAgent::readConfigElement(const json_t *element)
     }
 
     m_softwarecontainerWorkspace->m_enableWriteBuffer = wo;
+    m_containerConfig.setEnableWriteBuffer(wo);
```

Comment:
This is theoretically open for race conditions. It would be nicer if this parse method returned a config object that the caller could pass on. But I'm not sure it is worth fixing.